<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDCodeDialog</class>
 <widget class="QDialog" name="CreateDCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create DCode</string>
  </property>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Circle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Obround</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polygon</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBoxDCode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>D-Code number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="Circle">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxCircleDiameterPixel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixels</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxCircleDiameterMM">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Rectangle">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxRectangleWidthPixel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxRectangleWidthMM">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixels</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxRectangleHeightMM">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxRectangleHeightPixel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Obround">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxObroundWidthMM">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxObroundWidthPixel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxObroundHeightPixel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxObroundHeightMM">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Polygon">
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxPolygonDiameterPixel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxPolygonDiameterMM">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sides</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPolygonSides">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCreate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
